static/app/views/settings/projectPlugins/index.spec.jsx

```diff
@@ -48,31 +56,35 @@ describe('ProjectPluginsContainer', function () {
       url: `/projects/${org.slug}/${project.slug}/plugins/github/`,
       method: 'DELETE',
     });
-    wrapper = mountWithTheme(
-      <ProjectPlugins params={params} organization={organization} />
+    render(
+      <ProjectPluginsContainer
+        plugins={{plugins, loading: false}}
+        params={params}
+        organization={organization}
+      />
     );
   });
 
   it('calls `fetchPlugins` action creator after mount', function () {
     expect(fetchPlugins).toHaveBeenCalled();
   });
 
-  it('calls `enablePlugin` action creator when enabling plugin', function () {
-    const onChange = wrapper.find('ProjectPlugins').prop('onChange');
+  it('calls `enablePlugin` action creator when enabling plugin', async function () {
+    const button = (await screen.findAllByRole('checkbox'))[0];
 
     expect(enablePlugin).not.toHaveBeenCalled();
 
-    onChange('pluginId', true);
+    userEvent.click(button);
```

Comment:
should we look for something like `await screen.findByRole("checkbox", {name: "some name"})` ?

Reply:
unfortunately the checkbox is a little far away from the label in the DOM, but opted to query for the label using `findByText` and then querying the checkbox from that label's parent